Add request changes link to pages

bermudezmt opened the pull request:
Closes #1312 . @cnunciato - Hey Christian, can you please review this PR for styling issues? I'm not entirely sure why the `fa-check-square` icon isn't rendering properly.

UPDATE:

This is what it looks like after incorporating Christian's feedback:

![image](https://user-images.githubusercontent.com/5844484/61162843-7d55e200-a4bf-11e9-8dce-47531ceed7af.png)

justinvp commented on layouts/partials/docs/edit.html:
```diff
@@ -1,6 +1,15 @@
+      </a>
+   <li>
+      <a class="text-gray-600 hover:text-blue-700 text-xs" href="{{ $.Site.Params.repositoryURL }}/issues/new?body=File: [{{ .Path }}]({{ $.Page.Permalink }})">
+        <i class="fab fa-check-square mr-2"></i>
```
> How about maybe `fas fa-code-branch` for that one? @justinvp thoughts?

Actually, rather than `fas fa-code-branch`, how about `fas fa-edit` for the Edit icon?

cnunciato replied:
Works for me!

justinvp commented:
@chrsmith, Is the Travis build failing for this PR (`ERROR: TRAVIS_TOKEN is not set` when running `travisqueue start`) because @bermudezmt is submitting the PR from a fork of the repo, and it wouldn't have failed here if instead this was a PR from a feature branch in the `pulumi/docs` repo itself (named something like `bermudezmt/request-change`)? If that's the case, haven't we accepted 3rd party contributions to this repo from repo forks historically, with working PRs that wouldn't error during the Travis run?

joeduffy commented:
This is great!

bermudezmt commented:
I'm curious about the answer to failed checks, @justinvp. Thanks for yours and @cnunciato 's feedback -- I've addressed comments on spacing, capitalization, closing tags, and updated classes I think. Can one of you merge if all looks good? 

Or is standard practice for me to merge after 1 approval? Please let me know. :)

cnunciato commented:
It looks like we recently added [travisqueue](https://github.com/pulumi/travisqueue), which expects some `TRAVIS_*` environment variables to be present at build time, but Travis doesn't expose these variables to PR builds: https://docs.travis-ci.com/user/pull-requests/#pull-requests-and-security-restrictions

Do we need travisqueue here? Does [limiting concurrent builds](https://docs.travis-ci.com/user/customizing-the-build/#limiting-concurrent-jobs) not suffice?

chrsmith commented:
> Do we need `travisqueue` here? Does limiting concurrent builds not suffice?

We **always** need `travisqueue` and should **never** limit build concurrency. Otherwise things are "slow".

We've encountered this same issue in the past when adding `travisqueue` to the `docs` repo. I just introduced a regression. In the past we just skipped starting `travisqueue` if the env var wasn't set, which is probably a reasonable solution for the time being.

https://github.com/pulumi/docs/blob/6f81e92be803090881aa073a82d5acf71151a4da/scripts/travisqueue.sh
